Firebase/ios/Bonfire/Mobile/AppDelegate.swift

```diff
@@ -1,21 +1,21 @@
 import UIKit
 import Firebase
 import GoogleSignIn
-import RxSwift
 
 @UIApplicationMain
 final class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
-    let disposeBag = DisposeBag()
     var userService: UsersService!
 
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
         FIRApp.configure()
         GIDSignIn.sharedInstance().clientID = FIRApp.defaultApp()?.options.clientID
 
-//                try! FIRAuth.auth()?.signOut()
-//                GIDSignIn.sharedInstance().signOut()
+        // Uncomment these lines to sign out and start afresh!
```

Comment:
I was wondering - we have no logout functionality, so if you want to play with the app you might want to know how to do it?

I'm not precious about it though.